src/vs/platform/markers/common/problemMatcher.ts

```diff
@@ -1127,4 +1134,12 @@ registry.add('eslint-stylish', {
 	applyTo: ApplyToKind.allDocuments,
 	fileLocation: FileLocationKind.Absolute,
 	pattern: defaultPattern('eslint-stylish')
-});
\ No newline at end of file
+});
+
+registry.add('go', {
+	owner: 'typescript',
+	applyTo: ApplyToKind.allDocuments,
+	fileLocation: FileLocationKind.Relative,
+	filePrefix: '${cwd}',
```

Comment:
I chose `cwd` here over `workspaceRoot` so that the task can override the working directory if desired (with the default being the workspace root). Make sense?